clab/config/utils.go

```diff
@@ -101,35 +102,37 @@ func prepareLinkVars(lIdx int, link *types.Link, varsA, varsB Dict) error {
 		(varsB[vkFarEnd]).(Dict)[key] = v1
 	}
 
-	// Split all fields with a comma...
+	// Ensure values are added to both ends of the link
 	for k, v := range link.Vars {
-
-		r := SplitTrim(v)
-
 		if k == vkFarEnd || k == vkNodeName {
 			return fmt.Errorf("%s: reserved variable name '%s' found", link.String(), k)
 		}
 
-		if k == vkLinkIP && len(r) == 1 {
-			// calc the remote IP
-			ipF, err := ipFarEndS(v)
-			if err != nil {
-				return fmt.Errorf("%s: %s", link.String(), err)
+		vv := reflect.ValueOf(v)
+		if vv.Kind() == reflect.Slice || vv.Kind() == reflect.Array {
+			// Array/slice should contain 2 values, one for each end of the link
+			if vv.Len() != 2 {
+				return fmt.Errorf("%s: var %s should contain 2 elements, found %d: %v", link.String(), k, vv.Len(), v)
```

Comment:
let's expand `var` to `variable`? since this is a user-facing error, I think it is better to spell the var as a variable